### PR TITLE
Updated README.md and Added contribution guidelines

### PR DESCRIPTION
**Pull request by hassiahk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,234 @@
----
-title: Chef Transformer
-emoji: 🍲
-colorFrom: blue
-colorTo: red
-sdk: streamlit
-app_file: app.py
-pinned: false
----
```
Comment (m3hrdadfi) [inline README.md:9]: To have the same functionality with HF-Space, we need to have them.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,234 @@
...
+![avatar](images/../asset/images/chef-transformer.png)
```
Comment (m3hrdadfi) [inline README.md:1]: The banner is already attached to the repo, so there is no need, or at least not at this time

File: README.md [hunk 3]
```diff
@@ -1,17 +1,234 @@
...
+
+## Evaluation
+
+The following table summarizes the scores obtained by the **Chef Transformer**. Those marked as (*) are the baseline models.
+
+|      Model      | BLEU  |  WER  | COSIM | ROUGE-2 |
+| :-------------: | :---: | :---: | :---: | :-----: |
+|   Recipe1M+ *   | 0.844 | 0.786 | 0.589 |    -    |
+|   RecipeNLG *   | 0.866 | 0.751 | 0.666 |    -    |
+| ChefTransformer | 0.203 | 0.709 | 0.714 |  0.290  |
```
Comment (m3hrdadfi) [inline README.md:217]: I know this is non-sense, but for now, we need to remove the BLEU score column in order to find a way to calculate the correct score

**Review by m3hrdadfi — CHANGES_REQUESTED** [review]
@hassiahk, I've added a few comments. Please, modify the sections and push the changes.

File: README.md (hunk 1 above)
Comment (hassiahk) [reply README.md]: Done! :)

File: README.md (hunk 2 above)
Comment (hassiahk) [reply README.md]: Done! :)

File: README.md (hunk 3 above)
Comment (hassiahk) [reply README.md]: Sure, removed it for now. Will take a look as well :)

**Comment by hassiahk** [comment]
> @hassiahk, I've added a few comments. Please, modify the sections and push the changes.

Done @m3hrdadfi.

**Pull request merged**